CHANGELOG.md

```
@@ -1,1531 +1,1614 @@
+
+### Fixed
+
+- Disabled back button behaviour to open the Home deck.
+
+## [0.0.1]
+
+### Added
+
+- Created basic web app which launches YouTube TV.
+
+[Unreleased]: https://github.com/webosbrew/youtube-webos/compare/v0.3.1...HEAD
```

markdownlint-cli2 (0.23.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)